### DISCUSSION TITLE
Autocomplete field names in queries

### DISCUSSION THREAD

**Discussion by RobiABC** [dbeaver/dbeaver#20057] [Q&A]
Hi, since yesterday the autocomplete of field name while writing an SQL query does not work anymore. For table name autocomplete works, but for field name once I indicate the table to do the query on, it doesn't work. Has this happened to anyone?

**Comment by LonwoLonwo** [comment]
Hello @RobiABC 

Could you please provide a query example? 
Or maybe this is your case? https://github.com/dbeaver/dbeaver/issues/20032

**Reply by RobiABC** [reply]
Hi @LonwoLonwo, yes #20032 is exactly my case

**Reply by RobiABC** [reply]
![immagine](https://github.com/dbeaver/dbeaver/assets/81297616/51339fcb-fbdd-4bf1-a813-2dbba67e92cc)

After the WHERE I used to be offered the fields of the table, now no longer

**Reply by LonwoLonwo** [reply]
The fix is already in our code but will only be available with the next 23.1.0 release.

As a workaround for now - you can add aliases to tables.

**Reply by RobiABC** [reply]
Thank you for your answer, but no, I can't add aliases... 
I will wait for the new release. Can you give me an approximate time frame?

**Reply by LonwoLonwo** [reply]
You can try our Early Access version: https://dbeaver.io/files/ea/
Our official release, 23.1.0, will be on June 05, 2023.
